client/src/entrypoints/admin/page-editor.js

```diff
@@ -282,6 +374,7 @@ $(() => {
   initSlugCleaning();
   initErrorDetection();
   initKeyboardShortcuts();
+  initHiddenBreadcrumbs();
```

Comment:
I think this function is enough lines now to warrant its own file, you can add a `breadcrumbs.js` file inside `includes` without changing any of the code otherwise (just `export default initHiddenBreadcrumbs`, then import it here and use it just like you are now).